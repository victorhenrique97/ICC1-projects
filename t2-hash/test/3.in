2
10
52
39
19
80
26
33
18
99
66
73
