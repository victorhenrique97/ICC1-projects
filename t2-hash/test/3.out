0: 52 80 26 18 66
1: 39 19 33 99 73
